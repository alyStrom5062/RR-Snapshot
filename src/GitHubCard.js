import React from "react";
import { Card } from "react-bootstrap";

const GitHubCard = () => {
    return (
        <Card>
            <Card.Img variant="top" src="" alt="Photo" />
            <Card.Body>
                <Card.Title>GitHub Username</Card.Title>
                <Card.Text>
                    About me
                </Card.Text>
            </Card.Body>
        </Card>
    )
}

export default GitHubCard;