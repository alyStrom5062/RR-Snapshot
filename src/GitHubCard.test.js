import React from "react";
import { render } from '@testing-library/react';
import GitHubCard from "./GitHubCard";

test("renders Github Card component", () => {
    const { asFragment } = render(<GitHubCard />);
    expect(asFragment()).toMatchSnapshot();
});