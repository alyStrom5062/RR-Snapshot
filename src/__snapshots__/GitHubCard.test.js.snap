// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Github Card component 1`] = `
<DocumentFragment>
  <div
    class="card"
  >
    <img
      alt="Photo"
      class="card-img-top"
      src=""
    />
    <div
      class="card-body"
    >
      <div
        class="card-title h5"
      >
        GitHub Username
      </div>
      <p
        class="card-text"
      >
        About me
      </p>
    </div>
  </div>
</DocumentFragment>
`;
